resource-aggregate/events/resourceLinksSnapshotTaken.go

```diff
@@ -160,9 +160,9 @@ func (e *ResourceLinksSnapshotTaken) Handle(ctx context.Context, iter eventstore
 }
 
 func (e *ResourceLinksSnapshotTaken) HandleCommand(ctx context.Context, cmd aggregate.Command, newVersion uint64) ([]eventstore.Event, error) {
```

Comment:
Please fix bug "[BUG] AuditContect is null in ResourceLinksPublished events"

Reply:
will be done in separate PR